bin/gatherer.py

```diff
@@ -195,10 +195,16 @@ def setup(decoder):
 
         else:
             LOGGER.debug("Using posttroll for %s", section)
+            try:
+                duration = CONFIG.getfloat(section, "duration")
+            except NoOptionError:
+                duration = None
+
             granule_trigger = trigger.PostTrollTrigger(
                 collectors, terminator,
                 CONFIG.get(section, 'service').split(','),
                 CONFIG.get(section, 'topics').split(','),
+                duration,
```

Comment:
Maybe use `duration=duration` here so that it's evident it is a keyword argument, like the following arguments.